app/packages/utilities/src/schema.ts

```diff
@@ -1,3 +1,16 @@
+export const DETECTION_EMBEDDED_DOC_TYPE = "fiftyone.core.labels.Detection";
+export const DETECTIONS_EMBEDDED_DOC_TYPE = "fiftyone.core.labels.Detections";
+export const SEGMENTATION_EMBEDDED_DOC_TYPE =
+  "fiftyone.core.labels.Segmentation";
+export const HEATMAP_EMBEDDED_DOC_TYPE = "fiftyone.core.labels.Heatmap";
+
+export const DENSE_LABEL_EMBEDDED_DOC_TYPES = [
+  DETECTION_EMBEDDED_DOC_TYPE,
+  DETECTIONS_EMBEDDED_DOC_TYPE,
+  SEGMENTATION_EMBEDDED_DOC_TYPE,
+  HEATMAP_EMBEDDED_DOC_TYPE,
+];
```

Comment:
Duplicates values in `./constants`. As noted offline, I will follow up

Reply:
Sounds good.